README.md

```diff
@@ -10,6 +10,10 @@ Once the conda environment is created, activate it:
 
 `conda activate microsetta-public-api`
 
+Install qiime2:
+
+`conda install -c qiime2/label/r2020.2 qiime2 q2-types`
```

Comment:
what channel is that?

Reply:
Comes from the [Q2 environment files](https://raw.githubusercontent.com/qiime2/environment-files/master/2020.2/release/qiime2-2020.2-py36-osx-conda.yml). I think the thought was to use the channel pin the Q2 version to the current latest version. Thinking about it for half a second longer, I believe there is an easier way to do that... also will actually leave it unpinned for now, until it causes issues...
```suggestion
`conda install -c qiime2 qiime2 q2-types`
```